akka-stream/src/main/scala/akka/stream/scaladsl/Tcp.scala

```diff
@@ -86,9 +87,11 @@ object Tcp extends ExtensionId[Tcp] with ExtensionIdProvider {
     })
 }
 
+@silent("deprecated")
 final class Tcp(system: ExtendedActorSystem) extends akka.actor.Extension {
   import Tcp._
 
+  // FIXME how do we deal with this vs attributes?
```

Comment:
Is that field still needed? Seems OK to me: you couldn't customize this before, and you still can't - or am I missing something?

Reply:
It is used to read the bind timeout value below, I guess we could just read the attribute directly from settings though.

Reply:
Changed to read the config value directly instead.